llvm/lib/Target/AMDGPU/AMDGPUISelLowering.h

```diff
@@ -387,6 +387,16 @@ class AMDGPUTargetLowering : public TargetLowering {
   MVT getFenceOperandTy(const DataLayout &DL) const override {
     return MVT::i32;
   }
+
+  virtual bool hasMultiplePredicateRegisters(EVT VT) const override {
+    // FIXME: This is only partially true. If we have to do vector compares,
```

Comment:
AMDGPU part looks fine. In future we will probably want to address this FIXME by passing in a node, not just a type.